src/utils/db/queries.rs

```diff
@@ -283,3 +285,55 @@ fn input_table_name_for(aggregation_round: AggregationRound) -> &'static str {
         AggregationRound::Scheduler => "scheduler_witness_jobs_fri",
     }
 }
+
+pub async fn get_proof_time(
```

Comment:
Let's make the name of this function consistent with what it does. The current name would be ok for a function that returns the proof time for a given proof, and what it actually does is to provide proof times within a period of time.

I think `get_proof_times_within_period` would be a proper name for it, but we can think of others.